rql.go

```diff
@@ -197,13 +217,20 @@ func (p *Parser) ParseQuery(q *Query) (pr *Params, err error) {
 	return
 }
 
+func (p *Parser) GetFields() []*Field {
+	fields := make([]*Field, 0, len(p.fields))
+	for _, v := range p.fields {
+		fields = append(fields, v)
+	}
+	return fields
+}
```

Comment:
```suggestion
func (p *Parser) Fields() []*Field {
	fields := make([]*Field, 0, len(p.fields))
	for _, v := range p.fields {
		fields = append(fields, v)
	}
	return fields
}
```
Why this method is needed? If it's for testing, let's use the existing public API instead.